docs: updates Medicare cardholder enrollment documentation to include in-person enrollment (#3632)
cal-itp/benefits: docs/explanation/enrollment-pathways/medicare-cardholders.md

## Overview

> This use case describes a feature in [Cal-ITP Benefits](https://benefits.calitp.org) that allows eligible riders to verify their active participation in the Medicare program to receive reduced fares for transit when paying by contactless debit or credit card at participating transits operators in California.

Reviewer: Looks like a typo:

```suggestion
This use case describes a feature in [Cal-ITP Benefits](https://benefits.calitp.org) that allows eligible riders to verify their active participation in the Medicare program to receive reduced fares for transit when paying by contactless debit or credit card at participating transit operators in California.
```

**Actor:** A person who uses public transit in California. For benefit eligibility, a Medicare cardholder is a person who has enrolled in the Medicare program, has a Medicare.gov account, and is not deceased.

**Goal:** To verify a transit rider’s Medicare status so they receive reduced fares when paying by contactless debit or credit card.

> **Precondition:** The California transit operator offers fixed route service, has installed and tested validator hardware necessary to collect fares using open-loop payments on bus or rail lines, and the operator has a policy in place to offer a transit benefit to riders enrolled in Medicare.

Reviewer: A suggestion in line with removing the California reference in the opening paragraph, and another for making the sentence structure more consistent.

```suggestion
**Precondition:** The transit operator offers fixed route service, has installed and tested validator hardware necessary to collect fares using open-loop payments on bus or rail lines, and has a policy in place to offer a transit benefit to riders enrolled in Medicare.
```

## Demonstration

> Here's a video walkthrough of the rider self-service enrollment experience when choosing the Medicare cardholder enrollment pathway.

Reviewer: Is "rider" redundant with "self-service"?

```suggestion
Here's a video walkthrough of the self-service enrollment experience when choosing the Medicare cardholder enrollment pathway.
```

    participant Benefits as Cal-ITP Benefits
    participant IdG as Identity Gateway
    participant Medicare as Medicare.gov
    participant BBAPI as Blue Button API
    participant TransitProcessor as Transit processor
TransitRider->>Benefits: visits benefits.calitp.org
    activate Benefits
Benefits-->>IdG: begins identity verification
    activate IdG
TransitRider->>Medicare: account authentication
    activate Medicare
    Note over Medicare: Authenticated (Y/N)
Medicare-->>IdG: authentication confirmation
    deactivate Medicare
IdG-->>BBAPI: sends required PII
    activate BBAPI
    Note over BBAPI: Deceased (Y/N)
BBAPI-->>IdG: returns response
    deactivate BBAPI
IdG-->>Benefits: eligibility response
    deactivate IdG
Benefits-->>TransitProcessor: card registration start
    activate TransitProcessor
TransitRider->>TransitProcessor: provides debit or credit card details
TransitProcessor-->>Benefits: card registration confirmation
    deactivate TransitProcessor
    deactivate Benefits
    Note over Benefits: Successfull enrollment
```

1. The transit rider visits the web application at benefits.calitp.org in a browser on their desktop computer or mobile device.

> 1. The transit rider chooses the transit operator that serves an area where they want to ride public transit.

Reviewer: Not a change here, but I notice that the numbered steps in the written description don't exactly align with the numbers shown on the diagram. Not sure if it'd be useful to try and align them or not? Maybe more trouble than it's worth.

Author: Being that both the flow and the diagram have ten steps does indirectly propose they align, but each piece of content serves a different purpose. The diagram covers the technical steps while the flow describes steps in the UX. Thus, I'm okay if there isn't direct alignment in the steps.

1. The transit rider chooses to verify their eligibility as a Medicare cardholder.

1. The Cal-ITP Benefits app interfaces with the [California Department of Technology Identity Gateway](https://digitalidstrategy.cdt.ca.gov/primary-elements.html) (IdG) to verify rider identity and benefit eligibility.

1. The transit rider authenticates with their Medicare.gov account or, if they don’t have one, creates a Medicare.gov account using their Medicare card.

1. The transit rider consents to share information from their Medicare.gov account to verify their eligibility for a transit benefit.

> 1. The IdG uses the response provided by the [Blue Button API](https://bluebutton.cms.gov/) to determine the rider’s eligibility for a transit benefit.

Reviewer: Maybe a useful elaboration?

```suggestion
1. The IdG uses the response provided by the [Blue Button API provided by the Centers for Medicare and Medicaid services](https://bluebutton.cms.gov/) to determine the rider’s eligibility for a transit benefit.
```

Author: Let's leave this aspect of the documentation lean, as following the link provides that additional context.

1. The IdG then passes an eligibility response as Medicare enrollment status = TRUE to the Cal-ITP Benefits app to indicate the person is eligible for a benefit.

1. The transit rider provides the debit or credit card details they use to pay for transit to the [transit processor](../../index.md#transit-processors) that facilitates fare collection for the transit provider.

1. The app registers the transit rider’s debit or credit card for reduced fares.

> ## Alternative self-service flows

Reviewer: I get it, but this heading strikes me as a little odd. To me, it sorta implies "alternative flows that can still lead to a successful enrollment". It's sorta just a collection of miscellaneous usage notes. I can't come up with a better suggestion for the heading off the top of my head, though.

I also note that they are not _really_ specific to the Medicare pathway. I wonder if it'd make sense to reword them to be generic to all self-service flows (or at least all that go through IdG?) and put them in an include file that can be inserted on each enrollment pathway page.

Author: While perhaps it could be presented better, this section gets at flows that end in an exception or are edge cases from the "happy path." Let's leave them for now and iterate when we can improve it.

I'd like to discuss using includes here and in other places in the documentation. Once we get a good "flat file" in place, let's look for opportunities to optimize.

- Suppose the transit rider cannot authenticate with Medicare.gov, or will not create an account. In either case, the app cannot determine their Medicare status and they cannot enroll their contactless debit or credit card for a reduced fare.

- Suppose the CDT Identity Gateway returns Medicare enrollment status = FALSE. In that case, the Cal-ITP Benefits app will not allow the transit rider to register their contactless debit or credit card for a reduced fare.

- Suppose the debit or credit card expires or is canceled by the issuer. In that case, the transit rider must repeat the basic flow to register the new debit or credit card.

- If the transit rider uses more than one debit or credit card to pay for transit, they repeat the basic flow for each card.

> ## In-person enrollment

Reviewer: All of the "agency"s in this section should be "provider", yeah?

Author: Yeah. I'll do a final pass through the document to align terms.

Agency_Staff->>Benefits: verifies eligibility
    Note over Benefits: eligible (Y/N)
Fare_processor-->>Benefits: card registration form
    activate Fare_processor
Transit_Rider->>Fare_processor: enters debit or credit card details
Fare_processor-->>Benefits: card registration confirmation
    deactivate Fare_processor
    deactivate Benefits
    Note over Benefits: Successful enrollment
```

1. The transit rider visits an agency office or enrollment event in person.

1. A transit agency staff member logs into Cal-ITP Benefits Administrator, typically on a tablet device.

1. The transit agency staff member launches in-person enrollment and chooses Medicare cardholder as the eligibility type.

1. The transit rider hands the transit agency staff member their government-issued photo ID and Medicare card.

1. The transit agency staff member confirms the person’s identity and verifies that the Medicare card indicates the person is an active enrollee.

1. The transit agency staff member hands the transit rider the tablet so they can enter the debit or credit card details for the card they use to pay for transit.

> 1. The app registers the transit rider’s debit or credit card with the [transit processor](../../index.md#transit-processors).

Reviewer: Suggestion to make this more parallel with the self-service process description above:

```suggestion
1. The transit agency staff member hands the transit rider the tablet so they can enter the debit or credit card details for the card they use to pay for transit into the form provided by the [transit processor](../../index.md#transit-processors) that facilitates fare collection for the transit agency.

1. The app registers the transit rider’s debit or credit card for reduced fares.
```

- The transit rider doesn’t have to lock up funds on a closed-loop card offered by the transit operator.

> - The transit rider pays for transit rides with the same contactless debit or credit card they use for everyday purchases.

Reviewer: Hmm, the changes as proposed might imply that it _has_ to be the same card. It might be useful to combine these bullets.

I don't know why it's deciding not to let me make a suggestion here, but this is what I'd propose:

> - The transit rider doesn’t have to lock up funds on a closed-loop card offered by the transit operator. They can instead pay for transit rides with any contactless debit or credit card, just like they would for everyday purchases.

Author: They are distinct. The benefit of not having funds locked up on a closed loop card means the solution gives the person more financial flexibility in managing limited funds, while the other means that a person can pay for transit the same way they pay for everything else—no legacy or proprietary payment method required.

> - The transit rider can enroll in a transit benefit from home when it is convenient for them; they are not required to visit a transit agency office in person unless they choose the in-person enrollment method.

Reviewer: "they choose the in-person enrollment method" feels off to me, since we don't really present users with that choice. Maybe something like this?

```suggestion
- The transit rider can enroll in a transit benefit from home when it is convenient for them; they are not required to visit a transit agency office in person. And transit agencies can still offer in-person enrollment for riders who need or want that.
```

Author: By making a in-person enrollment a standard feature, we expect agencies to offer it in their marketing as an alternative to self-serve. We will also likely begin suggesting riders use in-person in the app copy when they reach certain error states.

Reviewer: That makes sense!
